The-New-Fork/openfood
# [Main] add get balance function and unittest

## Task
[JC-1908](https://thenewfork.atlassian.net/browse/JC-1908)

## Related Task
[JC-1907](https://thenewfork.atlassian.net/browse/JC-1907)

## Summary
add get_balance function and unittest

## Changes
- add explorer_get_balance_final function and divided by 100,000,000
- add unittest for explorer_get_balance_final function

## Others (Optional)
How to run test :
- Go to **test folder** (/deploy-samples/simple-working/organization/blocknotify-python/openfood_lib_dev/tests)
- run command **pytest -s test_explorer_lib.py::test_explorer_get_balance_final** (to check balance before and after convert)
- run command **pytest -v test_explorer_lib.py::test_explorer_get_balance_final** (to run test and to see that test has been passed)

If you want to run test for all function in test_explorer_lib.py, just use : **pytest -v test_explorer_lib.py**
